tests/discovery/test_hosts.py

```diff
@@ -70,3 +73,34 @@ def __init__(self, event):
 class testAzureMetadataApi(object):
     def __init__(self, event):
         assert config.azure
+
+
+class TestDiscoveryUtils:
+    @staticmethod
+    def test_cidr_discovery():
```

Comment:
Use a method name that describes the test case:
```suggestion
    def test_generate_hosts_valid_cidr():
```